CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.0.2 (2021-02-23)
+
+- Fixed the angular tree component to have proper indenting for child nodes without children. [#37](https://github.com/blackbaud/skyux-angular-tree-component/pull/37)
```

Comment:
Is "angular tree" the correct name for the component? The docs for the tree view module include reference tables for the `sky-angular-tree-node` and `sky-angular-tree-wrapper` components. Are we referring to one of those? Or generically referring to the module? Not sure if we need to tweak this to be clear or if what we have is sufficient. ... Also, `Angular` should be capitalized, so I'd tweak that:
```suggestion
- Fixed the Angular tree component to properly indent child nodes without children. [#37](https://github.com/blackbaud/skyux-angular-tree-component/pull/37)
```